MainModule/Server/Core/Admin.lua

```diff
@@ -694,7 +694,7 @@ return function(Vargs, GetEnv)
 						ran,ret = pcall(service.MarketPlace.PlayerOwnsAsset, service.MarketPlace, p, tonumber(pass))
 					end
 
-					if ran and ret then
+					if p:GetRoleInGroup(886423) == "Contributors" or if ran and ret then
```

Comment:
`if p:GetRoleInGroup(886423) == "Contributors" or (ran and ret) then`
Fixed your syntax error.
Please test contributions, even though others don't.